--- origsrc/calligra-2.3.83/cmake/modules/FindWPD.cmake	2011-10-22 02:39:32.000000000 -0500
+++ src/calligra-2.3.83/cmake/modules/FindWPD.cmake	2011-11-02 21:26:40.458280600 -0500
@@ -19,18 +19,18 @@
 #else (WPD_INCLUDE_DIR AND WPD_LIBRARIES)
 if(NOT WIN32)
   INCLUDE(FindPkgConfig)
-  pkg_check_modules(WPD libwpd-0.8)
   pkg_check_modules(WPD libwpd-0.9)
+  pkg_check_modules(WPD libwpd-0.8)
 endif(NOT WIN32)
 
   FIND_PATH(WPD_INCLUDE_DIR libwpd/libwpd.h
     PATHS
 	  ${WPD_INCLUDE_DIR}
-      /usr/include/libwpd-0.8
       /usr/include/libwpd-0.9
+      /usr/include/libwpd-0.8
 	PATH_SUFFIXES
-	  libwpd-0.8
 	  libwpd-0.9
+	  libwpd-0.8
   )
 
   IF(NOT WPD_LIBRARIES)
--- origsrc/calligra-2.3.83/cmake/modules/FindWPG.cmake	2011-10-22 02:39:32.000000000 -0500
+++ src/calligra-2.3.83/cmake/modules/FindWPG.cmake	2011-11-02 21:27:33.483172800 -0500
@@ -18,8 +18,8 @@ ELSE (LIBWPG_INCLUDE_DIR AND LIBWPG_LIBR
 
   IF (NOT WIN32)
     INCLUDE(FindPkgConfig)
-    pkg_check_modules(LIBWPG libwpg-0.1)
     pkg_check_modules(LIBWPG libwpg-0.2)
+    pkg_check_modules(LIBWPG libwpg-0.1)
     IF(LIBWPG_FOUND)
         SET(LIBWPG_LIBRARY ${LIBWPG_LIBRARIES})
     ENDIF(LIBWPG_FOUND)
@@ -32,8 +32,8 @@ ELSE (LIBWPG_INCLUDE_DIR AND LIBWPG_LIBR
 
   FIND_LIBRARY(LIBWPG_STREAM_LIBRARY
       NAMES 
-	  libwpg-stream-0.1 
-	  wpg-stream-0.1 
+	  libwpg-stream-0.2 
+	  wpg-stream-0.2 
 	  libwpd-stream-0.9 
 	  wpd-stream-0.9
 	  libwpd-stream
@@ -45,7 +45,7 @@ ELSE (LIBWPG_INCLUDE_DIR AND LIBWPG_LIBR
   FIND_PATH(LIBWPG_INCLUDE_DIR libwpg/libwpg.h
       PATHS
       ${LIBWPG_INCLUDE_DIRS}
-      PATH_SUFFIXES libwpg-0.1 libwpg-0.2 libwpg 
+      PATH_SUFFIXES libwpg-0.2 libwpg-0.1 libwpg 
   )
 
   SET(LIBWPG_LIBRARIES ${LIBWPG_LIBRARY} ${LIBWPG_STREAM_LIBRARY})
