--- origsrc/calligra-2.3.83/plan/plugins/filters/icalendar/CMakeLists.txt	2011-10-22 02:39:34.000000000 -0500
+++ src/calligra-2.3.83/plan/plugins/filters/icalendar/CMakeLists.txt	2011-11-02 16:48:07.519670600 -0500
@@ -2,7 +2,7 @@
 add_definitions(-DQT3_SUPPORT -DQT3_SUPPORT_WARNINGS)
 
 ##TODO revision?
-find_library(LIBKCAL_LIBRARY libkcal.so)
+find_library(LIBKCAL_LIBRARY kcal)
 if(LIBKCAL_LIBRARY)
     set(HAVE_LIBKCAL 1)
 endif(LIBKCAL_LIBRARY)
