--- origsrc/calligra-2.8.7/krita/sketch/draganddrop/CMakeLists.txt	2014-11-30 08:01:14.000000000 -0600
+++ src/calligra-2.8.7/krita/sketch/draganddrop/CMakeLists.txt	2015-01-21 01:59:02.769861100 -0600
@@ -17,7 +17,7 @@ include_directories(
 qt4_automoc(${declarativedragdrop_SRCS})
 
 
-kde4_add_library(draganddropplugin SHARED ${declarativedragdrop_SRCS})
+kde4_add_library(draganddropplugin MODULE ${declarativedragdrop_SRCS})
 target_link_libraries(draganddropplugin ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTDECLARATIVE_LIBRARY})
 
 install(TARGETS draganddropplugin DESTINATION ${CMAKE_INSTALL_LIBDIR}/calligra/imports/org/krita/draganddrop)
