--- origsrc/calligra-2.9.4/krita/CMakeLists.txt	2015-05-03 07:08:17.000000000 -0500
+++ src/calligra-2.9.4/krita/CMakeLists.txt	2015-05-12 00:26:14.935313300 -0500
@@ -172,7 +172,7 @@
 install(FILES  krita.action DESTINATION ${DATA_INSTALL_DIR}/krita/actions)
 install(FILES  krita.appdata.xml DESTINATION ${SHARE_INSTALL_PREFIX}/appdata/ )
 
-if (${QTVERSION} VERSION_GREATER 4.7.0 AND NOT APPLE AND HAVE_OPENGL)
+if (FALSE) # (${QTVERSION} VERSION_GREATER 4.7.0 AND NOT APPLE AND HAVE_OPENGL)
     add_subdirectory(sketch)
     add_subdirectory(gemini)
 endif ()
