--- origsrc/calligra-2.9.11/CMakeLists.txt	2017-02-03 14:09:13.939331900 -0600
+++ src/calligra-2.9.11/CMakeLists.txt	2017-02-05 04:31:40.925195300 -0600
@@ -979,6 +979,10 @@ calligra_disable_product(APP_ACTIVE "not
 calligra_disable_product(APP_CSTESTER "not wanted")
 calligra_disable_product(APP_DEVTOOLS "not wanted")
 calligra_disable_product(APP_GEMINI "not wanted")
+calligra_disable_product(APP_BRAINDUMP "not wanted")
+calligra_disable_product(APP_KEXI "not wanted")
+calligra_disable_product(APP_KRITA "not wanted")
+calligra_disable_product(PART_QTQUICK "not wanted")
 
 #############################################
 ####     Calculate buildable products    ####
