--- origsrc/calligra-3.0.1/cmake/modules/FindLibRCPS.cmake	2017-03-18 20:34:17.000000000 -0500
+++ src/calligra-3.0.1/cmake/modules/FindLibRCPS.cmake	2017-11-06 16:57:01.483631300 -0600
@@ -23,7 +23,7 @@ if(LIBRCPS_INCLUDE_DIR AND LIBRCPS_LIBRA
 endif()
 
 find_path(LIBRCPS_INCLUDE_DIR librcps.h)
-find_library(LIBRCPS_LIBRARIES librcps.so)
+find_library(LIBRCPS_LIBRARIES rcps)
 
 if(LIBRCPS_INCLUDE_DIR AND LIBRCPS_LIBRARIES)
   set(FIND_LIBRCPS_VERSION_SOURCE

