--- origsrc/calligra-3.0.1/libs/main/tests/CMakeLists.txt	2017-03-18 20:34:17.000000000 -0500
+++ src/calligra-3.0.1/libs/main/tests/CMakeLists.txt	2017-11-06 18:23:48.168639800 -0600
@@ -15,6 +15,7 @@ endmacro()
 
 set(priorityqueue_test_SRCS priorityqueue_test.cpp ../MainDebug.cpp)
 komain_add_unit_test(priorityQueueTest ${priorityqueue_test_SRCS}  LINK_LIBRARIES KF5::I18n Qt5::Test)
+set_target_properties(priorityQueueTest PROPERTIES COMPILE_DEFINITIONS "KOMAIN_STATIC_DEFINE")
 
 ########### next target ###############
 
--- origsrc/calligra-3.0.1/plugins/formulashape/tests/CMakeLists.txt	2017-03-18 20:34:17.000000000 -0500
+++ src/calligra-3.0.1/plugins/formulashape/tests/CMakeLists.txt	2017-11-07 13:44:29.062065000 -0600
@@ -51,7 +51,7 @@ TestCursor.cpp
    ../KoFormulaTool.cpp
    ../FormulaCommand.cpp
 )
-if(NOT WIN32)
+if(NOT WIN32 AND NOT CYGWIN)
 set(TestCursor_SRCS
     ${TestCursor_SRCS}
     ../FormulaCursor.cpp)
